packages/react-components/react-textarea/src/stories/TextareaAppearance.stories.tsx

```diff
@@ -23,18 +32,34 @@ export const Appearance = () => {
   const styles = useStyles();
 
   return (
-    <div className={styles.container}>
+    <div className={styles.base}>
       <div>
         <Label htmlFor={outlineId}>Textarea with Outline appearance.</Label>
```

Comment:
I've been giving labels an extra left padding so they visually align with the components they label.